src/Compilers/CSharp/Test/Syntax/Parsing/RefReadonlyReturnsTests.cs

```diff
@@ -138,9 +138,6 @@ static void Use<T>(T dummy)
 ";
             var comp = CreateCompilationWithMscorlib45(text, new[] { ValueTupleRef, SystemRuntimeFacadeRef });
             comp.VerifyDiagnostics(
```

Comment:
in that case, we should remove the first line of the method. it is covered by others, and it is not proving the "unexpected bind time" here.